operators/trivy-operator/0.16.3/manifests/trivy-operator.v0.16.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,328 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: trivy-operator.v0.16.3
+  namespace: trivy-operator
+  annotations:
+    capabilities: Basic Install
+    categories: Security
+    description: Keeps security report resources updated
+    certified: "false"
+    containerImage: docker.io/aquasec/trivy-operator:0.16.3
+    createdAt: 2023-18-17T25:17:20Z
```

Comment:
I think the CI is complaining about the validity of this timestamp.

Reply:
thanks checking now